game/theater/caucasus.py

```diff
@@ -1,8 +1,44 @@
 from game.theater.projections import TransverseMercator
```

Comment:
These are generated by export_coordinates.py, so this will be clobbered if this ever needs to re-run. Sorry, I should have generated a comment that says that.  Put these things directly in `CausasusTheater` and co.

Reply:
Will fix.